proto/streamlit/proto/NewReport.proto

```diff
@@ -95,13 +95,12 @@ message CustomThemeConfig {
     MONOSPACE = 2;
   }
 
-  string name = 1;
-  string primary_color = 2;
-  string secondary_color = 3;
-  string secondary_background_color = 4;
-  string background_color = 5;
-  string text_color = 6;
-  FontFamily font = 7;
+  string primary_color = 1;
+  string secondary_color = 2;
+  string secondary_background_color = 3;
+  string background_color = 4;
+  string text_color = 5;
+  FontFamily font = 6;
```

Comment:
`font` is very broad and it has a ton of constituents. `font_family`?

Reply:
Hm, I'll likely keep this as-is for now since we actually changed the name from `font_family` -> `font` after some input from product. The proto could always use a different internal name than the config option that we expose to the user, but I think the trouble of conversion back and forth cancels out a lot of the benefits of the more descriptive name